lib/raw_net_capture.rb

```diff
@@ -24,7 +24,7 @@ def reset
 end
 
 class RawHTTPCapture < StringIO
-  attr_reader :raw_received, :raw_sent
+  attr_reader :raw_received, :raw_sent, :transactions
```

Comment:
We should just dump the `raw_sent` and `raw_received` terminology and go whole hog on request, response for this class